packages/amplify-category-api/src/provider-utils/awscloudformation/utils/graphql-schema-utils.ts

```diff
@@ -13,12 +14,14 @@ export const writeSchemaFile = (pathToSchemaFile: string, schemaString: string)
 
 export const generateRDSSchema = async (
   context: $TSContext,
-  databaseConfig:
-  ImportedDataSourceConfig, pathToSchemaFile: string
+  databaseConfig: ImportedDataSourceConfig,
+  pathToSchemaFile: string,
 ): Promise<string> => {
   // Establish the connection
   let adapter: DataSourceAdapter;
   let schema: Schema;
+  const UNABLE_TO_CONNECT_MESSAGE = 'Failed to connect to the specified RDS Data Source. Check the connection details in the schema and re-try. Use "amplify api update-secrets" to update the user credentials.';
```

Comment:
No need to redefine this constant in the method itself.

Reply:
It's just a style n.p. not super important, but for constants like this that don't change, if you hoist out of the function you're not redeclaring it on each function invocation.

Alternately, you could just inline the string if it's only used in a single place, but I don't have a particularly strong opinion here.